### DISCUSSION TITLE
Appearance InputRange

### DISCUSSION THREAD

**Discussion by loxy** [taiga-family/taiga-ui#1270] [Q&A] [answered]
Hi!

Is it somehow possible to apply appearance on `InputRange`? Seems not the case, right?

Best regards,
Kersten

**Comment by waterplea** [comment, marked as answer]
Yes, it appears to be hardcoded. You can make a PR to add token then, like with the rest of controls.

**Reply by loxy** [reply]
Ok, I will do that. Thank you!

**Reply by loxy** [reply]
Done in https://github.com/Tinkoff/taiga-ui/pull/1280
